<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>DNA4Collections by DNA4Collections</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>DNA4Collections</h1>
        <p>Web repository for the 2015 DNA4Collections hackathon</p>
        <img src="images/footprint-30879-small.png" alt="logo" />
        <!-- LINKS TO OTHER RESOURCES GO HERE -->

        <!--p class="view"><a href="https://github.com/DNA4Collections/web">View the Project on GitHub <small>DNA4Collections/web</small></a></p>


        <ul>
          <li><a href="https://github.com/DNA4Collections/web/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/DNA4Collections/web/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/DNA4Collections/web">View On <strong>GitHub</strong></a></li>
        </ul-->
      </header>
      <section>
      <h1>Pre-invitation for hackathon #DNA4Collections</h1>
        <p>Many institutions in the Health and Life Sciences maintain collections of biological specimens for a variety of reasons. Among these are gene banks in plant and animal breeding, environmental samples for monitoring purposes, specimen collections for cataloguing biodiversity, and patient samples in biomedical research. </p>

<p>Collection material can be a source of DNA. It is usually not easy to get sufficient DNA out of a sample that has not been collected for the explicit purpose of storing that kind of molecules. Recent developments in DNA sequencing technologies, and particularly expected developments in so called ‘single molecule sequencing’, enable access to precious samples collected over the course of hundreds of years. DNA is nature’s ultimate information-carrying molecule. DNA contains a wealth of information on the natural history of organisms and the molecular functions that define that organism.  However, because DNA information is so easily digitized, it can at the same time be used to make the collections more accessible. DNA information from every organism, from bacteria to whales, can be digitized (sequenced), stored, and searched in the same way. Due to the dropping costs of high-throughput DNA sequencing, more and more of these collection specimens are selected for (re-)sequencing. This raises considerable methodological challenges, among which are:</p>

<ul>
<li>Handling degraded and fragmented DNA from old specimens</li>
<li>The prevalence of non-model organisms and dearth of reference genomes</li>
<li>Mixtures of DNA from different species in the data, either due to sampling method, post-mortem contamination, or infection by pathogens</li>
</ul>

<p>In addition, the transformation of institutes that maintain such collections into accidental sequencing centers raises many practical challenges in terms of capacity for:</p>

<ul>
<li>Data storage, data retention and data publishing</li>
<li>HPC for de novo assembly and annotation</li>
<li>State-of-the-art bioinformatics methods development</li>
</ul>

<p>The prospect of generating and maintaining large datastores of thousands, perhaps in the future millions, of collection samples offers novel ways in which we can start mining that data. This offers novel conceptual and practical challenges, that, interestingly, share similarities to metagenomics.</p>

<ul>
<li>Collection material faces similar challenges as are encountered in metagenome studies. The material might be contaminated with fungal and bacterial DNA, some of it of a much more recent origin than the sample itself. As with metagenome studies, providing context is important. </li>
<li>Filling in the phylogenetic and, for species, temporal (some of these collections are old!) and geographic gaps, which is particularly important for species in regions where collecting is difficult because the original habitat disappeared, the species died out (locally or entirely) or because of legislative restrictions for sampling.</li>
<li>De novo assembly based on degraded material will always be a challenge, but connecting to databases with reference genomes of the same or related species will allow making of guided assemblies as an alternative.</li>
</ul>

<p><strong>To address these methodological and practical challenges from a technology perspective we are organizing a 3-4 day hackathon, entitled #DNA4Collections, slated for April 2015 (exact date and location TBA).</strong></p>

<p><strong>Who should take part? How to prepare?</strong>
A hackathon is an event where programmers and other stakeholders engage in intensive, collaborative software development over a short period of time. Prospective participants of this hackathon are invited to consider specific challenges within the scope described above that could, to a first approximation, be addressed collaboratively within a week. Once invitations start going out a process of active participation and engagement will commence that will drive self-organization into subgroups around these challenges.</p>

<p><strong>Interested?</strong>
Send a mail message to <a href="mailto:dna4collections@dtls.nl">dna4collections@dtls.nl</a> and we will make sure to keep you posted.</p>
      </section>
      <footer>
        <!-- LOGOS OF ORGANIZATIONS GO HERE -->
        <p>This project is maintained by <a href="https://github.com/DNA4Collections">DNA4Collections</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
